draft-ietf-wish-whip.md

```diff
@@ -266,8 +266,7 @@ WHIP clients generating the HTTP PATCH body with the SDP fragment and its subseq
  - As per {{!RFC9429}}, only m-sections not marked as bundle-only can gather ICE candidates, so given that the "max-bundle" policy is being used, the SDP fragment will contain only the offerer-tagged m-line of the bundle group.
  - The WHIP client MAY exclude ICE candidates from the HTTP PATCH body if they have already been confirmed by the WHIP session with a successful HTTP response to a previous HTTP PATCH request.
 
-If the WHIP session is using entity-tags for identifying the ICE sessions in explained in {{http-patch-usage}}, a WHIP client sending a PATCH request for performing trickle ICE MUST include an "If-Match" header field with the latest known entity-tag as per {{Section 13.1.1 of !RFC9110}}.
-When the PATCH request is received by the WHIP session, it MUST compare the indicated entity-tag value with the current entity-tag of the resource as per {{Section 13.1.1 of !RFC9110}} and return a "412 Precondition Failed" response if they do not match. If the HTTP PATCH request does not contain an "If-Match" header the WHIP session MUST return an "428 Precondition Required" response as per {{Section 3 of !RFC6585}}.
+WHIP sessions that support Trickle ICE MUST return a strong ETag as per {{Section 8.8.3 of !RFC9110}} in a successful response to a POST to the WHIP endpoint. WHIP clients that support Trickle ICE MUST use the ETag as per {{Section 13.1.1 of !RFC9110}} in its PATCH requests. Missing our outdated ETags in the PATCH requests from WHIP clients will be answered by WHIP sessions as per {{Section 13.1.1 of !RFC9110}} and {{Section 3 of !RFC6585}}.
```

Comment:
I think s/our/or , right?

Just to be clear, I am not against clarifying that 412 or 428 error codes are used in these cases, but there should be no BCP14 text respecifying (so I am against the MUST in the original text). Given that, what about this change to the second sentence, in the proposed text:

> Missing or outdated ETags in the PATCH requests from WHIP clients will be answered by WHIP sessions as per {{Section 13.1.1 of !RFC9110}} and {{Section 3 of !RFC6585}}, with a "428 Precondition Required" response for a missing entity tag, and a "412 Precondition Failed" response for a non-matching entity tag.

I think this should cover the existing text, without respecifying what's already defined.